Celeste.Mod.mm/Patches/Monocle/Engine.cs

```diff
@@ -59,12 +58,16 @@ public patch_Engine(int width, int height, int windowWidth, int windowHeight, st
         protected override extern void Update(GameTime gameTime);
 
         private static float GetTimeRateComponentMultiplier(Scene scene) {
-            return scene?.Tracker.GetComponents<TimeRateModifier>()
-                                 .Cast<TimeRateModifier>()
-                                 .Where(trm => trm.Enabled)
-                                 .Select(trm => trm.Multiplier)
-                                 .Aggregate(1f, (acc, val) => acc * val)
-                   ?? 1;
+            if (scene == null)
+                return 1f;
+
+            float result = 1f;
+            foreach (TimeRateModifier trm in scene.Tracker.GetComponents<TimeRateModifier>()) {
+                if (trm.Enabled)
+                    result *= trm.Multiplier;
+            }
+
+            return result;
```

Comment:
uhhh, this seems unrelated to the PR? It is a good change and I approve of it regardless.